vault_program/src/initialize_vault_update_state_tracker.rs

```diff
@@ -75,12 +75,28 @@ pub fn process_initialize_vault_update_state_tracker(
         &vault_update_state_tracker_seeds,
     )?;
 
-    let additional_assets_need_unstaking = vault
-        .calculate_additional_supported_assets_needed_to_unstake(
+    let additional_assets_need_unstaking = if vault.additional_assets_need_unstaking() > 0 {
+        // indicates an old partially cranked state
+        // no vault operations can be done until a full update cycle has run
+        // this means that the `additional_assets_need_unstaking` will not change
+        // between epochs and we can effectively carry on where we left off
+        vault.additional_assets_need_unstaking()
+    } else {
+        // Update fees to new values
+        // Fees can only be updated here so `additional_assets_need_unstaking` will be static
+        // Otherwise there may be a mismatch between withdrawn assets and outstanding claim tickets
+        vault.set_withdrawal_fee_bps(vault.next_withdrawal_fee_bps());
```

Comment:
do we need this here if fee changes need an up-to-date vault?

Reply:
discussed offline - this is okay for now